internal/status/updater_test.go

```diff
@@ -162,7 +162,7 @@ var _ = Describe("Updater", func() {
 									{
 										Type:               string(v1beta1.ListenerConditionReady),
 										Status:             status,
-										ObservedGeneration: 123,
+										ObservedGeneration: 0,
```

Comment:
would it make sense to use non-default (non-zero) value to make sure that the field is set correctly?

Reply:
updated